### DISCUSSION TITLE
gns3-server 2.2.17 and gns3-gui 2.2.17 using python3.9

### DISCUSSION THREAD

**Discussion by jnatschev** [GNS3/gns3-gui#3111] [General]
hi.

i'd like to report that i have been successful in getting gns3-server 2.2.17 and gns3-gui 2.2.17 working using python3.9 (3.9.1 using conda channel conda-forge) on macos 11.0.1 (big sur) and interacting with gns3-vm 2.2.17 on vmware fusion 12.1.0 with a topology of csr1000v routers of version 16.12.4a. i was able to telnet to the console of each router and configure basic layer3 connectivity and bgp routing.

reporting in the event that it is of interest.

**Comment by grossmj** [comment]
Thanks for the information! :+1:
